css-fonts-5/Overview.bs

```diff
@@ -180,6 +180,11 @@ The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors d
 <a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively. The `size-adjust`
 descriptor defines a scalar by which glyph outlines and metrics from this font are multiplied.
 
+Both values accepted by 'ascent-override', 'descent-override', and 'line-gap-override' measure
+distance in the block direction. When the text is <a href="#typeset-sideways">typeset sideways</a>,
```

Comment:
Aw, crap, this isn't right either. I have to add in a clause "typeset sideways or in horizontal writing mode"